includes/filerepo/file/LocalFile.php

```diff
@@ -350,6 +350,18 @@ function loadFromRow( $row, $prefix = 'img_' ) {
 			$this->$name = $value;
 		}
 
+		/**
+		 * Check, how often is this code executed. Scope: the following if block.
+		 *
+		 * @author Mix
+		 * @see SUS-809
+		 */
+		\Wikia\Logger\WikiaLogger::instance()->debugSampled(
+			0.01,
+			'SUS-809',
+			[ 'method' => __METHOD__, 'exception' => new Exception() ]
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "exception" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)